Lib/ast.py

```diff
@@ -689,5 +692,23 @@ def main(args=None):
     print(dump(tree, include_attributes=args.include_attributes,
                indent=args.indent, show_empty=args.show_empty))
 
+_deprecated = {
+        'slice': globals().pop("slice"),
```

Comment:
Note this means `from ast import *` will no longer pick these up. That's probably OK for 3.15 but let's mention it in docs.